pokemongo_bot/cell_workers/sniper.py

```diff
@@ -152,10 +154,25 @@ def validate(self):
             raise
             
     def _fixname(self,name):
-        name = name.replace("mr-mime","mr. mime")
-        name = name.replace("farfetchd","farfetch'd")
+        if name:
+            name = name.replace("mr-mime","mr. mime")
+            name = name.replace("farfetchd","farfetch'd")
+            name = name.replace("Nidoran\u2642","nidoran m")
+            name = name.replace("Nidoran\u2640","nidoran f")
         return name
 
+    def _get_closest_name(self, name):
```

Comment:
Again, read the description. Difflib is already used in pokemonoptimizer to TRY and get a real pokemon name from a supplied string. Eg. if source is "pigeot", it *should* return "pidgeot"... in many cases, it seems to work well.